contrib/sssd.sysusers

```diff
@@ -1,2 +1 @@
-#Type Name   ID  GECOS               Home directory  Shell
```

Comment:
If you are to remove this line, then do it in the first commit to avoid another commit.

Reply:
I squashed the commits.